_rules/html-has-lang-b5c3f8.md

```diff
@@ -25,24 +25,21 @@ The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s)
 
 ## Expectation
 
-Each test target has a `lang` or `xml:lang` attribute that is neither empty ("") nor only [whitespace](#whitespace).
-
-**Note**: HTML5 recommends using `lang` instead of `xml:lang`. This is not known to impact accessibility, which is why use of both is permitted by this rule.
+Each test target has a `lang` attribute that is neither empty ("") nor only [whitespace](#whitespace).
```

Comment:
Shouldn't this be ASCII whitespace (https://infra.spec.whatwg.org/#ascii-whitespace) or is that a different pull request?

Reply:
Gah, I forgot to change it 🤦‍♂